.travis.yml

```diff
@@ -4,13 +4,9 @@
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
Also Windows is still pending. So ping me when that releases and I can pull that too.